source/_patterns/components/view-selector/view-selector.hbs

```diff
@@ -0,0 +1,11 @@
+<div class="pf-view-selector" role="group" aria-label="View selector">
```

Comment:
I wasn't sure. Since the buttons are btn-links - no border etc - it doesn't do much for us - it doesn't reduce the additional css we need, but it does add some positioning. I debated about it and decided since each atom/element should be placed by its parent, it wasn't giving me much of anything useful, though also not really harmful. But I'm fine with adding it if you think I should.